atomic_reactor/plugins/pre_resolve_module_compose.py

```diff
@@ -137,9 +142,25 @@ def _resolve_compose(self):
         # since we our interaction with the PDC is read-only, this is fine for our needs and
         # makes things simpler.
         pdc_client = PDCClient(server=self.pdc_url, ssl_verify=not self.pdc_insecure, develop=True)
+        self.read_configs_general()
+
+        modules = self.data.get('modules', [])
+        if modules:
+            source_spec = modules[0]
+            if len(modules) > 1:
+                self.log.info("compose config contains multiple modules,"
+                              "using first module %s", source_spec)
+        else:
+            raise RuntimeError('"compose" config is missing "modules", required for Flatpak')
```

Comment:
Could rephrase this like:
```
if not modules:
    raise RuntimeError(...)

source_spec = modules[0]
...
```

to avoid unnecessary indentation.

Reply:
ok